do-restore.go

```diff
@@ -23,28 +23,137 @@ import (
 	"io/ioutil"
 	"os"
 	"strings"
+	"regexp"
+	"encoding/json"
+	"github.com/grafana-tools/sdk"
```

Comment:
Let split external import clauses from internal with a space line? It is not a strict requirement of course just minor style notify. I suppose it makes import statement more readable.

Reply:
_That's_ why I see so much code with a space in the imports. I was unaware of that convention. That makes sense.